src/test/lib/operators/export_test.cpp

```diff
@@ -99,6 +93,16 @@ TEST_P(OperatorsExportMultiFileTypeTest, ExportWithoutFileType) {
   EXPECT_TRUE(compare_files(reference_filename, filename));
 }
 
+TEST_P(OperatorsExportMultiFileTypeTest, NameAndDescription) {
+  const auto exporter = std::make_shared<Export>(nullptr, "a/file/path", GetParam());
+  const auto file_type_string = GetParam() == FileType::Binary ? "binary" : "csv";
```

Comment:
Should the type of a variable be in the variable`s name?

Reply:
Also do not really like it. However, we have a `FileType` enum and this is a string represenation of the enum value (which we have to create for getting a lowercase string)... Do you have any better suggestion?